README.md

```diff
@@ -29,62 +24,60 @@ Build the standard way.
 go build github.com/giantswarm/aws-operator
 ```
 
-
 ## Running aws-operator
 
 See [this guide][examples-local].
 
 [examples-local]: https://github.com/giantswarm/aws-operator/blob/master/examples
 
-
 ## Architecture
 
+The operator uses our [operatorkit](1) framework. It manages an `awsconfig`
+CRD using a generated client stored in our [apiextensions](2) repo. Releases
+are versioned using [version bundles](3).
 
-### S3 buckets
+The operator provisions guest Kubernetes clusters running on AWS. It runs in a
+host Kubernetes cluster also running on AWS. 
 
-S3 buckets are used for storing cloudconfigs, which are fetched by the EC2
-instances and executed. Then cloud-init, by using those cloudconfigs, takes
-care of running the needed systemd services and containers. The result of
-successful execution of cloud-init on each EC2 instance should be the working
-Kubernetes cluster.
+[1]:https://github.com/giantswarm/operatorkit
+[2]:https://github.com/giantswarm/apiextensions
+[3]:https://github.com/giantswarm/versionbundle
 
-Buckets are created for each customer, then inside the buckets there are
-folders for each cluster.
+### CloudFormation
 
-Each cluster contains the cloudconfig for master and worker instances.
+The guest Kubernetes clusters are provisioned using [AWS CloudFormation](4). The
+resources are split between 3 CloudFormation stacks.
 
-For example, assuming that we have the following customers with clusters:
+* guest-main manages the guest cluster resources.
+* host-setup manages an IAM role used for VPC peering.
```

Comment:
Maybe good to mention here or above that some of the resources are created in the host account, for `guest-main` it's clear that it is created in the guest, but maybe not that the others are created in the host.

Reply:
Good point, I've added a note on this.